Automattic/vip-go-mu-plugins
# Uploads: Add length check for file name/path

To provide clearer errors when the full path (`/wp-content/uploads/path/to/.../file.png`) is too long.

Runs the checks early in the process (`wp_handle_upload_prefilter`) since the `wp_check_filetype_and_ext` hook is late and doesn't allow the ability to customize errors returned to the user.

See PLAT-1162

## TODO

- [x] Remove tests for removed functions
- [ ] Check if we need to run through unique filename endpoint?

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change works and has been tested locally (or has an appropriate fallback).

## Steps to Test

1. Create a file with a name that's 231 characters or more.
1. Try to upload.
1. It should fail.
1. Upload a file with a shorter name; it should upload fine.
